concent-builder/configure.yml

```diff
@@ -91,7 +92,8 @@
 
         - name:    Install yasha in global python environment using pip
           pip:
-            name: yasha
+            executable: pip3
+            name:       yasha
```

Comment:
Your commit description is wrong. You're not modifying yasha. You're modifying concent-builder playbooks to use yasha on Python 3 instead of Python 2.

Reply:
Done.